0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -460,12 +461,17 @@ ILP Address prefix of the ledger that this transfer is going through on.
 #### amount
 <code>**amount**:String</code>
 
-A decimal amount, represented as a string. MUST be positive. The supported precision is defined by each ledger plugin and can be queried by the host via [`getInfo`](#getinfo). The ledger plugin MUST throw an `InsufficientPrecisionError` if the given amount exceeds the supported level of precision.
+An integer amount, represented as a string. MUST be positive. The supported precision is defined by each ledger plugin and can be queried by the host via [`getInfo`](#getinfo). The ledger plugin MUST throw an `InsufficientPrecisionError` if the given amount exceeds the supported level of precision.
+
+#### ilp
+<code>**ilp**:String</code>
+
+An [ILP header](../0003-interledger-protocol/), denoting the payment's final destination.
 
 #### data
```

Comment:
Nothing right now, and it should probably be removed. I do think there might be a place for having a place to put data that is meant for the next connector but isn't necessarily added by the original sender and therefore can't be in the ILP packet. (Not sure what I would call that though - we can probably come up with something better than `data`?)

For example: Let's say as a connector I want next-hop connectors to notify me when they've prepared their outgoing transfer. Maybe I want to use that information for risk management purposes or whatever. So it would be nice if I could attach something to the transfer which says "please notify me when you've prepared this".

Perhaps this new field can be just another buffer and we define the exact format later?